[1.62.0] Target/Minimum KOTS version reference docs (#662)
replicatedhq/kots.io: content/reference/v1beta1/application.md

### yAxisTemplate
Y axis labels template. Use `{{ value }}`.

## targetKotsVersion
The KOTS version that is targeted by the release.

> Specifying this in the application spec of the release will enforce compatibility checks for new installations and block the installation if the KOTS version being used is greater than the targeted KOTS version.

Reviewer: On the initial install, does it not check for an exact version match?

Author: no, it will check if it's less than or equal to the target version.

Author: target kots version was originally "max kots version" but we renamed it because it doesn't really block app updates (and for other reasons too), so it makes sense this way.

For channels that have the release active, the install command for existing clusters - which displayed on the channel card - will include that specific version of KOTS.

> Specifying a target KOTS version will not prevent an end user from upgrading to a higher version of KOTS after the initial installation. Similarly, if a new version of the application specifies a higher target KOTS version than what is currently installed, the end user will not be prevented from deploying that version of the application. Instead, if an end-user's admin console is running a version of KOTS that is less than the target version specified in the version of the application that they have deployed, a message will be displayed in the footer of the admin console to indicate that a newer supported version of KOTS is available.

Reviewer: Made changes here. Take a look.

> Specifying a target KOTS version will not prevent an end user from upgrading to a higher version of KOTS after the initial installation. Similarly, if a new version of the application specifies a higher target KOTS version than what is currently installed, the end user will not be prevented from deploying that version of the application. Instead, if an end-user's admin console is running a version of KOTS that is less than the target version specified in the version of the application that they have deployed, a message will be displayed in the footer of the admin console to indicate that a newer supported version of KOTS is available.

Author: ```suggestion
Specifying a target KOTS version will not prevent an end user from upgrading to a higher version of KOTS after the initial installation. Similarly, if a new version of the application specifies a higher target KOTS version than what is currently installed, the end user will not be prevented from deploying that version of the application. Instead, if an end-user's admin console is running a version of KOTS that is less than the target version specified in the new version of the application, a message will be displayed in the footer of the admin console to indicate that a newer supported version of KOTS is available.
```

Reviewer: What does new mean here? Do they have to have this version installed or can it just be in their version history?

### yAxisTemplate
Y axis labels template. Use `{{ value }}`.

## targetKotsVersion
The KOTS version that is targeted by the release.

> Specifying this in the application spec of the release will enforce compatibility checks for new installations and block the installation if the KOTS version being used is greater than the targetted KOTS version.

Reviewer: On the initial install, does it not check for an exact version match?

> For channels that have the release active, the install command for existing clusters - which displayed on the channel card - will include that specific version of KOTS.

Reviewer: The wording at the beginning of this paragraph confuses me. Is this still what you're trying to say?
```suggestion
If the latest release in a channel specifies a target KOTS version, the install command for existing clusters—which is displayed on the channel card in the vendor portal—will be modified to install that specific version of KOTS.
```

> If the latest release in a channel specifies a target KOTS version, the install command for existing clusters—which is displayed on the channel card in the vendor portal—will be modified to install that specific version of KOTS.

Reviewer 2: ```suggestion
If the latest release in a channel specifies a target KOTS version, the install command for existing clusters will be modified to install that specific version of KOTS. The install command for existing clusters can be found on the channel card in the [vendor portal](https://vendor.replicated.com). 
```

> Specifying a target KOTS version will not prevent an end user from upgrading to a higher version of KOTS after the initial installation. Similarly, if a new version of the application specifies a higher target KOTS version than what is currently installed, the end user will not be prevented from deploying that version of the application. Instead, if an end-user's admin console is running a version of KOTS that is less than the target version specified in the new version of the application, a message will be displayed in the footer of the admin console to indicate that a newer supported version of KOTS is available.

Reviewer 2: ```suggestion
Specifying a target KOTS version will not prevent an end user from upgrading to a higher version of KOTS after the initial installation. Similarly, if a new version of the application specifies a higher target KOTS version than what is currently installed, the end user will not be prevented from deploying that version of the application. If an end-user's admin console is running a version of KOTS that is less than the target version specified in the new version of the application, a message will be displayed in the footer of the admin console to indicate that a newer supported version of KOTS is available.
```

> For embedded cluster installs, it is important to keep the version of the [KOTS add-on](https://kurl.sh/docs/add-ons/kotsadm) in sync with the target KOTS version specified in the application spec. In particular, if the KOTS add-on version is greater than the target KOTS version, the initial installation will fail.

Reviewer 2: ```suggestion
For embedded cluster installs, it is important to keep the version of the [KOTS add-on](https://kurl.sh/docs/add-ons/kotsadm) in sync with the target KOTS version specified in the application spec. If the KOTS add-on version is greater than the target KOTS version, the initial installation will fail.
```

Specifying a target KOTS version will not prevent an end user from upgrading to a higher version of KOTS after the initial installation. Similarly, if a new version of the application specifies a higher target KOTS version than what is currently installed, the end user will not be prevented from deploying that version of the application. If an end-user's admin console is running a version of KOTS that is less than the target version specified in the new version of the application, a message will be displayed in the footer of the admin console to indicate that a newer supported version of KOTS is available.

For embedded cluster installs, it is important to keep the version of the [KOTS add-on](https://kurl.sh/docs/add-ons/kotsadm) in sync with the target KOTS version specified in the application spec. If the KOTS add-on version is greater than the target KOTS version, the initial installation will fail.

## minKotsVersion (Beta)
The minimum KOTS version that is required by the release.

> Specifying this in the application spec of the release will enforce compatibility checks for both new installations and application updates, and will block if the KOTS version being used is less than the minimum KOTS version.

Reviewer: ```suggestion
Specifying this in the application spec of the release will enforce compatibility checks for both new installations and application updates, and will block if the currently deployed KOTS version is less than the specified minimum KOTS version.
```
